modules/light-clients/09-localhost/light_client_module_test.go

```diff
@@ -0,0 +1,320 @@
+package localhost_test
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	clienttypes "github.com/cosmos/ibc-go/v8/modules/core/02-client/types"
+	connectiontypes "github.com/cosmos/ibc-go/v8/modules/core/03-connection/types"
+	channeltypes "github.com/cosmos/ibc-go/v8/modules/core/04-channel/types"
+	commitmenttypes "github.com/cosmos/ibc-go/v8/modules/core/23-commitment/types"
+	host "github.com/cosmos/ibc-go/v8/modules/core/24-host"
+	"github.com/cosmos/ibc-go/v8/modules/core/exported"
+	localhost "github.com/cosmos/ibc-go/v8/modules/light-clients/09-localhost"
+	ibctesting "github.com/cosmos/ibc-go/v8/testing"
+	"github.com/cosmos/ibc-go/v8/testing/mock"
+)
+
+func (suite *LocalhostTestSuite) TestStatus() {
+	lightClientModule, found := suite.chain.GetSimApp().IBCKeeper.ClientKeeper.GetRouter().GetRoute(exported.LocalhostClientID)
+	suite.Require().True(found)
+	suite.Require().Equal(exported.Active, lightClientModule.Status(suite.chain.GetContext(), exported.LocalhostClientID))
+}
+
+func (suite *LocalhostTestSuite) TestVerifyMembership() {
+	var (
+		path  exported.Path
+		value []byte
+	)
+
+	testCases := []struct {
+		name     string
+		malleate func()
+		expPass  bool
```

Comment:
i guess eventually we can change these to `expError`

Reply:
can be addressed in https://github.com/cosmos/ibc-go/issues/5959